frontend/projects/upgrade/src/app/features/dashboard/feature-flags/modals/upsert-feature-flag-modal/upsert-feature-flag-modal.component.html

```diff
@@ -19,8 +19,17 @@
       <mat-label class="ft-14-400">Key</mat-label>
       <input matInput formControlName="key" placeholder="Key for this feature flag" class="ft-14-400"/>
       <mat-hint class="form-hint ft-12-400">
-        {{'feature-flags.upsert-flag-modal.key-hint.text' | translate}}
-        <a class="learn-more-link" href="https://www.upgradeplatform.org/" target="_blank" rel="noopener noreferrer">Learn more</a>
+        <ng-container *ngIf="!validationError; else duplicateError">
```

Comment:
please remove code that isn't being used now since we are saying snackbar is place for message for now